src/Themes/NexusMods.Themes.NexusFluentDark/Styles/Controls/Icons/IconsStyles.axaml

```diff
@@ -1,490 +1,347 @@
+            <icons:UnifiedIcon Classes="ClearAll" />
+            <icons:UnifiedIcon Classes="Cog" />
+            <icons:UnifiedIcon Classes="Download" />
+            <icons:UnifiedIcon Classes="HelpCircle" />
+            <icons:UnifiedIcon Classes="Help" />
+            <icons:UnifiedIcon Classes="History" />
+            <icons:UnifiedIcon Classes="Play" />
+            <icons:UnifiedIcon Classes="PlusCircleOutline" />
+            <icons:UnifiedIcon Classes="ProgressDownload" />
+            <icons:UnifiedIcon Classes="PauseCircleOutline" />
+            <icons:UnifiedIcon Classes="PauseCircleFilled" />
+            <icons:UnifiedIcon Classes="RemoveCircleOutline" />
+            <icons:UnifiedIcon Classes="Star" />
+            <icons:UnifiedIcon Classes="TrashCanOutline" />
+            <icons:UnifiedIcon Classes="PlayCircleOutline" />
+            <icons:UnifiedIcon Classes="PlayCircleFilled" />
+            <icons:UnifiedIcon Classes="Image" />
+            <icons:UnifiedIcon Classes="Description" />
+            <icons:UnifiedIcon Classes="Tune" />
+            <icons:UnifiedIcon Classes="ArrowLeft" />
+            <icons:UnifiedIcon Classes="ArrowRight" />
+            <icons:UnifiedIcon Classes="Plus" />
+            <icons:UnifiedIcon Classes="MenuDown" />
+            <icons:UnifiedIcon Classes="MenuUp" />
+            <icons:UnifiedIcon Classes="Close" />
+            <icons:UnifiedIcon Classes="OpenInNew" />
+            <icons:UnifiedIcon Classes="Tab" />
+            <icons:UnifiedIcon Classes="ChevronLeft" />
+            <icons:UnifiedIcon Classes="ChevronRight" />
+            <icons:UnifiedIcon Classes="FolderOutline" />
+            <icons:UnifiedIcon Classes="FolderOpenOutline" />
+            <icons:UnifiedIcon Classes="File" />
+            <icons:UnifiedIcon Classes="Check" />
+            <icons:UnifiedIcon Classes="DeleteOutline" />
+            <icons:UnifiedIcon Classes="DragHorizontal" />
+            <icons:UnifiedIcon Classes="DragVertical" />
+            <icons:UnifiedIcon Classes="Search" />
+            <icons:UnifiedIcon Classes="Home" />
+            <icons:UnifiedIcon Classes="ToggleSwitch" />
+            <icons:UnifiedIcon Classes="ToggleSwitchOff" />
+            <icons:UnifiedIcon Classes="Game" />
+            <icons:UnifiedIcon Classes="Undo" />
+            <icons:UnifiedIcon Classes="Redo" />
+            <icons:UnifiedIcon Classes="AddCircle" />
+            <icons:UnifiedIcon Classes="AlertCircle" />
+            <icons:UnifiedIcon Classes="AlertOctagon" />
+            <icons:UnifiedIcon Classes="MonitorDiagnostics" />
+            <icons:UnifiedIcon Classes="DiagnosticPage"/>
+            <icons:UnifiedIcon Classes="FileDocumentOutline" />
+            <icons:UnifiedIcon Classes="MusicNote" />
+            <icons:UnifiedIcon Classes="VideoOutline" />
+            <icons:UnifiedIcon Classes="Refresh" />
+            <icons:UnifiedIcon Classes="Collections" />
```

Comment:
You are missing 
- News
- Bookmark
- Eye
- ListBulleted

You added 
- Collections
I guess the first three were not used and the last one was renamed?

Reply:
The ones I removed weren't used anywhere. We have a couple more icons that are unused, but those were just too annoying to work with.